scripts/sizeSnapshot/webpack.config.js

```diff
@@ -72,62 +72,63 @@ async function getWebpackEntries() {
       // WARNING: Changing the name will break tracking of bundle size over time
       // If the name of the package changes, rename its display name in https://github.com/eps1lon/mui-contributor-dashboard/blob/main/src/pages/SizeComparison.tsx
       id: '@material-ui/core',
-      path: path.join(path.relative(workspaceRoot, materialPackagePath), 'index.js'),
+      import: '@mui/material',
     },
     ...materialComponents,
     {
       id: '@material-ui/lab',
-      path: path.join(path.relative(workspaceRoot, labPackagePath), 'index.js'),
+      import: '@mui/lab',
     },
     ...labComponents,
     {
       id: '@material-ui/styles',
-      path: 'packages/mui-styles/build/index.js',
+      import: '@mui/styles',
     },
     {
       id: '@material-ui/private-theming',
-      path: 'packages/mui-private-theming/build/index.js',
+      import: '@mui/private-theming',
     },
     {
       id: '@material-ui/system',
-      path: 'packages/mui-system/build/index.js',
+      import: '@mui/system',
     },
     {
       id: 'createBox',
-      path: 'packages/mui-system/build/createBox/index.js',
+      import: '@mui/system/createBox',
     },
     {
       id: 'createStyled',
-      path: 'packages/mui-system/build/createStyled/index.js',
+      import: '@mui/system/createStyled',
     },
     {
       id: '@material-ui/core/styles/createTheme',
-      path: 'packages/mui-material/build/styles/createTheme.js',
```

Comment:
This import will be disallowed in ESM so I'm adding the possibility to import a specific export